src/app/plugins/githubuploadadapter.js

```diff
@@ -129,10 +129,14 @@ export class Adapter {
 						this._sendRequest( data );
 					} ) )
 					.then( ( /* { file, response } */ ) => {
-						// Upload concluded! Simply send the file URL back, according to CKEditor specs.
-						return {
-							default: returnUrl
-						};
+						// Step 3: Retrieve the final uploaded asset URL using response data gathered on Step 1.
+						return this.sendAssetRequest( assetResponse )
```

Comment:
Let the `sendAssetRequest` be a private method, just like the others around here. No need to expose it.